src/main/kotlin/net/irisshaders/lilybot/database/DatabaseManager.kt

```diff
@@ -56,6 +64,7 @@ object DatabaseManager {
 
 		transaction {
 			SchemaUtils.createMissingTablesAndColumns(Warn)
+			SchemaUtils.createMissingTablesAndColumns(Components)
```

Comment:
Can this method not take in multiple parameters?

Reply:
Fixed, thanks.